goflags.go

```diff
@@ -19,6 +19,16 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
+const (
```

Comment:
These error constants should instead be written in form of errors.

```
var ErrMarshal = errors.New("could not unmarshal configuration file")
```

For errors that require more context, I guess we can create custom error types.

Reply:
Done